src-tauri/src/command/subcommand.rs

```diff
@@ -0,0 +1,287 @@
+use serde::Serialize;
+use std::{
+    fs::{self, File},
+    io::Write,
+    path::PathBuf,
+    sync::{Arc, Mutex},
+};
+
+use super::errors::{CommandError, Result};
+
+use crate::{
+    base::{Link, OpenGraph, Score, Scores},
+    cli, cli_unknown_arg, init_statics, Cli, ARK_SHELF_WORKING_DIR, METADATA_PATH, PREVIEWS_PATH,
```

Comment:
`METADATA_PATH` is from older arklib version, new constant is called `METADATA_STORAGE_FOLDER`

But `PROPERTIES_STORAGE_FOLDER` should be used instead anyway.

Metadata is always generated by resource content, i.e. it is merely cache for some computed attributes. It's not defined for bookmarks, yet. At this moment, metadata is only used with images and videos in Navigator app.